ui/public/i18n/language.json

```diff
@@ -167,7 +167,8 @@
     "shutdown_ok": "The system is shutting down...",
     "shutdown_error": "Shutting down error occured.",
     "hostname_change": "After hostname change you have to login again",
-    "hostname_invalid_warn": "The server FQDN must not contain 'localhost' or 'localdomain'. You have to change your machine hostname before proceeding with other configurations"
+    "hostname_invalid_warn": "The server FQDN must not contain 'localhost' or 'localdomain'. You have to change your machine hostname before proceeding with other configurations",
```

Comment:
I can't see the difference, please dot not add this on the patch

Reply:
I've added alias string (removed from other PR). It's not correct?

Reply:
The `alias` translation is already inserted in the file: https://github.com/NethServer/nethserver-cockpit/blob/725b01d6b6234bc38575c8cfba3e83705f7b9e66/ui/public/i18n/language.json#L139 It's a duplicated string

Reply:
Ops sorry. I didn't see it. I will remove it in a minute.